File: 백준/Bronze/17094. Serious Problem/Serious Problem.c
#include<stdio.h>
#include<string.h>
int main(void){
    int n;
    scanf("%d",&n);
    char s[100005];
    scanf("%s",s);

    int cnt2 = 0;
    int cnte = 0;

    for(int i = 0; i < n; i++){
        if(s[i] == 'e')
            cnte++;
        else if(s[i] == '2')
            cnt2++;
    }
    if(cnte > cnt2)
        printf("e");
    else if(cnte < cnt2)
        printf("2");
    else
        printf("yee");

    return 0;
}

File: 백준/Bronze/17094. Serious Problem/Serious Problem.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        String s = sc.next();

        int cnte = 0;
        int cnt2 = 0;

        for(int i = 0; i < n; i++){
            char c = s.charAt(i);
            if(c == '2')
                cnt2++;
            else if(c == 'e')
                cnte++;
        }

        if(cnt2 > cnte)
            System.out.println("2");
        else if(cnt2 < cnte)
            System.out.println("e");
        else
            System.out.println("yee");

    }
}